lib/kredis/types/list.rb

```diff
@@ -24,4 +25,13 @@ def append(*elements, pipeline: nil)
   def clear
     del
   end
+
+  private
+
+    def default
```

Comment:
This has side effects, so needs a different name. Like `initialize_with_defaults`.